core-web/libs/portlets/dot-ema/src/lib/edit-ema-editor/components/edit-ema-palette/components/edit-ema-palette-content-type/edit-ema-palette-content-type.component.ts

```diff
@@ -0,0 +1,48 @@
+import { JsonPipe, NgFor, NgIf } from '@angular/common';
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, Output } from '@angular/core';
+import { FormControl, ReactiveFormsModule } from '@angular/forms';
+
+import { InputTextModule } from 'primeng/inputtext';
+
+import { DotIconModule, DotMessagePipe, DotSpinnerModule } from '@dotcms/ui';
+
+import { EditEmaPaletteStoreStatus } from '../../shared/edit-ema-palette.enums';
+
+@Component({
+    selector: 'dot-edit-ema-palette-content-type',
+    standalone: true,
+    imports: [
+        NgFor,
+        NgIf,
+        ReactiveFormsModule,
+        DotIconModule,
+        JsonPipe,
```

Comment:
Are we using this?

Reply:
Yes! to format the attribute data-item on drag